### PR TITLE
feat(adminmanual): Add section about CalDAV rate limits

### PR DESCRIPTION
**Pull request by ChristophWurst**
### ☑️ Resolves

* For https://github.com/nextcloud/server/pull/43732

### 🖼️ Screenshots

Bildschirmfoto vom 2024-02-22 09-30-51

### CODE REVIEW

**Review by tcitworld — APPROVED** [review]

File: admin_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -141,3 +141,22 @@ Known backends
 ~~~~~~~~~~~~~~
 
 * `Calendar Resource Management <https://github.com/nextcloud/calendar_resource_management>`_: database backend with CLI configuration for admins
```
Comment (mnMomma) [inline admin_manual/groupware/calendar.rst:143]: How do I configure this

Comment (ChristophWurst) [reply admin_manual/groupware/calendar.rst:143]: https://help.nextcloud.com/

**Pull request merged**